src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -1228,10 +1232,11 @@ private void bindConstantToProperty(final String propertyLabel, final Properties
    * @param pkg - class name to use as key
    * @return reflections.
    */
-  public static Reflections getReflectionsClass(final String pkg) {
+  public static List<Class<?>> getReflectionsClass(final String pkg)
```

Comment:
This probably needs to be renamed (and the Javadoc revised), as we're no longer dealing with Reflections.